draft/draft-ietf-netmod-syslog-model.xml

```diff
@@ -432,62 +432,94 @@ the following registration is requested:</t>
+    proper protection can have a negative effect on network operations
+    and on network security.
+  </t>
+
+  <t>
+    In addition there are data nodes that require careful analysis and
       review. These are the subtrees and data nodes and their
-      sensitivity/vulnerability: <list hangIndent="6" style="hanging">
-<t hangText="   facility-filter/pattern-match:">When writing this
-          node, implementations MUST ensure that the regular expression
-          pattern match is not constructed to cause a regular expression
-          denial of service attack due to a pattern that causes the regular
-          expression implementation to work very slowly (exponentially related
-          to input size).</t>
-
-<t hangText="   remote/destination/signing/cert-signer:">When
-          writing this subtree, implementations MUST NOT specify a private key
-          that is used for any other purpose.</t>
-</list>
-</t>
+      sensitivity/vulnerability:
+
+      <list hangIndent="6" style="hanging">
+	<t hangText=" facility-filter/pattern-match:">When writing
+	this node, implementations MUST ensure that the regular
+	expression pattern match is not constructed to cause a regular
+	expression denial of service attack due to a pattern that
+	causes the regular expression implementation to work very
+	slowly (exponentially related to input size).
+	</t>
+
+	<t hangText=" remote/destination/signing/cert-signer:">When
+	writing this subtree, implementations MUST NOT specify a
+	private key that is used for any other purpose.</t>
+      </list>
+  </t>
 
-<t>Some of the readable data nodes in this YANG module may be considered
-      sensitive or vulnerable in some network environments. It is thus
-      important to control read access (e.g., via get, get-config, or
-      notification) to these data nodes. These are the subtrees and data nodes
-      and their sensitivity/vulnerability: <list hangIndent="6" style="hanging">
-<t hangText="   remote/destination/transport:">This subtree contains
-          information about other hosts in the network, and the TLS transport
-          certificate properties if TLS is selected as the transport
-          protocol.</t>
-
-<t hangText="   remote/destination/signing:">This subtree contains
-          information about the syslog message signing properties including
-          signing certificate information.</t>
-</list>
-</t>
+  <t>
+    Some of the readable data nodes in this YANG module may be
+    considered sensitive or vulnerable in some network
+    environments. It is thus important to control read access (e.g.,
+    via get, get-config, or notification) to these data nodes. These
+    are the subtrees and data nodes and their
+    sensitivity/vulnerability:
+
+    <list hangIndent="6" style="hanging">
+      <t hangText=" remote/destination/transport:">This subtree
+      contains information about other hosts in the network, and the
```

Comment:
Maybe also mention that this subtree includes information about _services_ available on specific hosts in the network.  My point being that in addition to knowing that a.b.c.d exists in the network, you know it provides a syslog service, say, on udp/514.  And thus you might spam it (if it wasn't properly protected).